Minor edits (#120)
ferrous-systems/rust-exercises: exercise-book/src/simple-db.md

> 4. Empty payloads are allowed. In this case, the command is

Reviewer: ```suggestion
4. An empty payload is allowed. The command to publish an empty payload is
```

Author: I think something more like 

>  An empty payload is allowed is only allowed for the `PUBLISH\n` command..

Author: Or perhaps

> The PUBLISH command can have an empty payload, RETRIEVE must always have a non empty payload.

Reviewer: RETRIEVE doesn't have a payload.

    `PUBLISH \n`.

> Errors in the form or properties of the messages are

Reviewer: I think this got mangled?

Author: I was intentionally writing around violations, but we can either rewrite or igore.

Reviewer: ```suggestion
Issues with the format (or other properties) of the messages are
```

perhaps?
